# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vte/no-TIOCGPTPEER.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vte-0.59.0/src/pty.cc.vanilla	2020-02-20 22:14:44.443097259 +0000
+++ vte-0.59.0/src/pty.cc	2020-02-20 22:15:02.507097165 +0000
@@ -159,7 +159,7 @@
         auto const fd_flags = int{O_RDWR | ((priv->flags & VTE_PTY_NO_CTTY) ? O_NOCTTY : 0)};
         auto fd = int{-1};
 
-#ifdef __linux__
+#if 0
         fd = ioctl(masterfd, TIOCGPTPEER, fd_flags);
         /* Note: According to the kernel's own tests (tools/testing/selftests/filesystems/devpts_pts.c),
          * the error returned when the running kernel does not support this ioctl should be EINVAL.
